src/InteractiveWindow/Editor/IInteractiveWindowOperations2.cs

```diff
@@ -8,5 +8,10 @@ public interface IInteractiveWindowOperations2 : IInteractiveWindowOperations
         /// Copies the current selection to the clipboard.
         /// </summary>
         void Copy();
+
+        /// <summary>
+        /// Handles the user pressing any alphanumeric key.
+        /// </summary>
+        void TypeChar();
```

Comment:
Should this be called OnTypeChar (or OnBeforeTypeChar) since the implementation doesn't actually handle the character?